### DISCUSSION TITLE
AspCore and dependency injection

### DISCUSSION THREAD

**Discussion by AhmedSuror** [ctolkien/Slugify#134] [Q&A] [answered]
There is no documentation about using this Lib for slugifying AspCore HTTP requests, also there is no mentioning of Dependency Injection implementation.
Does this Lib provide both?

**Comment by ctolkien** [comment, marked as answer]
Hi @AhmedSuror  - sorry this one was missed. You can certainly just register an `(I)SlugHelper` in your DI container. If all of your configuration is always the same, it should be fine being a singleton.
